frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/portlets/components/ComponentPortlet.java

```diff
@@ -2227,6 +2234,7 @@ public void applyFilters(ActionRequest request, ActionResponse response) throws
         }
         response.setRenderParameter(PortalConstants.DATE_RANGE, nullToEmpty(request.getParameter(PortalConstants.DATE_RANGE)));
         response.setRenderParameter(PortalConstants.END_DATE, nullToEmpty(request.getParameter(PortalConstants.END_DATE)));
+        response.setRenderParameter("exactMatchCheckBox", nullToEmpty(request.getParameter("exactMatchCheckBox")));
```

Comment:
please create a constant in `PortalConstants.java` class and use it everywhere instead of `"exactMatchCheckBox"`.